web/src/lib/domain-logic/filterFundings.ts

```diff
@@ -58,26 +58,28 @@ export const filterFundings = (fundings: Funding[], business: Business): Funding
       return false;
     }
 
-    if (it.certifications !== undefined && it.certifications !== null) {
-      if (it.certifications.length > 0 && business.profileData.ownershipTypeIds.length > 0) {
-        const ownershipTypeIds = new Set(arrayOfOwnershipTypes.map((ownershipType) => ownershipType.id));
-        const ownershipTypeCerts = it.certifications.filter((cert) => ownershipTypeIds.has(cert));
+    if (
+      it.certifications &&
```

Comment:
[dust] We're doing this same check on `line 79`. I might consider encapsulating both these `if`s in one instance of this conditional check.